### PR TITLE
fix: MacOS getTitle() - segmentation fault

### PR DESCRIPTION
**Pull request by ahkohd**
Hi, @sentialx 

When **`getTitle()`** is invoked; It runs into `segementaion fault` on MacOS. 

After walking the code I found out the culprit line:
https://github.com/sentialx/node-window-manager/blob/8dd45f2dc2ce2293b9b632c41d01f7b776fb3da1/lib/macos.mm#L195
for some reasons `kCGWindowName` fails, so I use `kCGWindowOwnerName` instead.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -53,5 +53,17 @@
     "mutex": "cpp",
     "ratio": "cpp",
     "thread": "cpp"
-  }
+  },
+  "workbench.colorCustomizations": {
```
Comment (sentialx) [inline .vscode/settings.json]: I think it's not needed.

**Comment by sentialx** [comment]
@ahkohd Does it work on latest macOS 10.15?

File: .vscode/settings.json (hunk 1 above)
Comment (ahkohd) [reply .vscode/settings.json]: Yeah! that is Peacock's doing, I'll remove it.

**Comment by ahkohd** [comment]
> @ahkohd Does it work on latest macOS 10.15?

Yes! I am using macOS 10.15.1

I have prepared a screenshot for confirmation:

Screenshot 2020-06-04 at 16 12 23

**Comment by sentialx** [comment]
Thanks 👍

**Comment by ahkohd** [comment]
> Thanks 👍

Anytime! 👍

**Pull request merged**